Dockerfile

```diff
@@ -1,14 +1,18 @@
 FROM nginx:1.10
 MAINTAINER Wyatt Pearsall<Wyatt.Pearsall@cfpb.gov>
 
-RUN mkdir -p /usr/src/app
+RUN if [ -z ${SKIP_JS_BUILD+x} ]; then echo "Installing JS deps" && apt-get update && \
+    apt-get install -y curl g++ git make && \
```

Comment:
We can remove `git` from here. That was required because of USWDS and the latest release from them fixed it.